Mailbox.Tests/DataLoaderTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using Newtonsoft.Json;
+using System.IO;
+using System.Collections.Generic;
+
+namespace Mailbox.Tests
+{
+    [TestClass]
+    public class DataLoaderTests
+    {
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void DataLoader_CreatWithNullStream()
+        {
+            DataLoader data = new DataLoader(null);
+            data.Dispose();
+        }
+
+        [TestMethod]
+        public void DataLoader_Save_Success()
+        {
+            MemoryStream memoryStream = new MemoryStream();
```

Comment:
MemoryStream is `IDisposable` and should be properly cleaned up.
```suggestion
            using MemoryStream memoryStream = new MemoryStream();
```